Apps/TimelineApp/gradle.properties

```diff
@@ -18,4 +18,5 @@ android.useAndroidX=true
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1Ijoia3MyMTgyIiwiYSI6ImNsenJzNjRxcjI0c3QyanFyaDF5a21kZ2gifQ.wTr6vSBv9TEbmT1R3oaMH
```

Comment:
this is the secret key and should not be exposed. Please rotate your secret key, and set it in your PC's local gradle.properties. `settings.gradle` line24 already handle it. 

https://docs.mapbox.com/android/maps/guides/install/#configure-your-secret-token